From f6194e605c3b3bebe0b6c78139b6552a3f28f8b4 Mon Sep 17 00:00:00 2001
From: Valentin David <me@valentindavid.com>
Date: Sat, 24 Jan 2026 12:13:27 +0100
Subject: [PATCH 7/7] lvm2: Disable event activation by default

Because lvm2 does not have a distro wide configuration in /usr and all
is in /etc, configuring it is not compatible with systemd-sysupdate
OSes - because systemd-udevd starts before systemd-tmpfiles. Also on
GNOME OS, installation is confusing for the user that cannot format a
disk before they deactivate the volume groups taking phyisical volumes
from that disk.

The solution is to disable this by default

Now, Bluefin users will be fine with lvm2 installations, as
bootc/OSTree still provides /etc at the right time. So for those
distros we will set the initial value in /etc to activate the groups.

It does not seem Endless OS uses lvm2 volume groups. Furthermore, it
could also set the configuration in /etc since it uses OSTree.
---
 elements/components/lvm2-base.bst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/elements/components/lvm2-base.bst b/elements/components/lvm2-base.bst
index 5f289a56a..5b24218eb 100644
--- a/elements/components/lvm2-base.bst
+++ b/elements/components/lvm2-base.bst
@@ -22,6 +22,7 @@ variables:
     --with-staticdir=%{sbindir}
     --with-usrlibdir=%{libdir}
     --with-usrsbindir=%{sbindir}
+    --with-default-event-activation=0
 
 config:
   configure-commands:
-- 
2.52.0

